cli/src/main.rs

```diff
@@ -300,10 +300,20 @@ fn main() -> Result<(), Box<dyn error::Error>> {
         Arg::with_name("commitment")
             .long("commitment")
             .takes_value(true)
-            .possible_values(&["recent", "single", "singleGossip", "root", "max"])
+            .possible_values(&[
+                "processed",
+                "confirmed",
+                "finalized",
+                "recent",
+                "single",
+                "singleGossip",
+                "root",
+                "max",
```

Comment:
Maybe a code comment that mentions these four are deprecated and when for future refactoring efforts